internal/pkg/install/prepare.go

```diff
@@ -178,6 +179,11 @@ func Prepare(data *userdata.UserData) (err error) {
 			continue
 		}
 
+		if label == constants.BootPartitionLabel {
+			if err = syslinux.Prepare(device); err != nil {
```

Comment:
During development I found that writing `gptmbr.bin` to `/dev/sda` would fail. Not fail as in the go code thew an error, but failed in that the first 400 some odd bytes of `/dev/sda` didn't have the epexcted contents.